### ISSUE TITLE
Uninitialized Constant Error with Capistrano 3.10.1

### ISSUE THREAD

**Issue by JosephLeedy** [davidalger/capistrano-magento2#92]
I updated to Capistrano 3.10.1 from 3.10.0 and I am getting the following error now:

```
The deploy has failed with an error: Exception while executing as [redacted]: uninitialized constant Capistrano::Magento2::Setup::DateTime
```
I am using version 0.7.2 of the `capistrano-magento2` gem.

**Comment by JosephLeedy** [comment]
I just downgraded to 3.10.0 and I am getting the same error.

**Comment by czettnersandor** [comment]
Hi, I solved this for you. I think you updated ruby, not capistrano as this is nothing to do with capistrano version. The issue here is the DateTime object requires the 'date', so adding it on top of the magento2.rb solves this issue.

I'm using ruby 2.3.1p112

I'm not sure if this is the correct way of solving it, but you can use it until @davidalger comes up with a better solution.

**Comment by davidalger** [comment]
Hey guys! Just merged in the PR provided by @czettnersandor. It does fix this issue. Based on my research, I believe this only affected Ruby 2.3 and later (previous versions made DateTime available without using require).

The fix has been published as a part of v0.7.3.

**Comment by JosephLeedy** [comment]
Thanks, David! I owe you a beer at Imagine.
